voluptuous/humanize.py

```diff
@@ -0,0 +1,39 @@
+from voluptuous import Invalid, MultipleInvalid
+from voluptuous.error import Error
+
+
+MAX_VALIDATION_ERROR_ITEM_LENGTH = 500
+
+
+def _nested_getitem(data, path):
+    for item_name in path:
+        try:
```

Comment:
I'd probably lean against the `reduce` option even though it's nice and concise, for python3 compatibility (though I know it's not officially supported)